designer/server/src/main/scala/pl/touk/nussknacker/ui/api/SettingsResources.scala

```diff
@@ -87,6 +88,16 @@ object DeploymentCommentSettings {
     new DeploymentCommentSettings(validationPattern, exampleComment)
   }
 
+  def createNonEmptyDeploymentCommentSettings(
```

Comment:
It is used only in tests. Also, in tests use `DeploymentCommentSettings.unsafe` instead of `create`, you do not need to go through all that Valid/Invalid matching.